guides/installation/setups/symfony-cli.md

```diff
@@ -0,0 +1,180 @@
+---
+nav:
+  title: Symfony CLI
+  position: 10
+
+---
+
+# Symfony CLI
+
+Symfony CLI is a popular tool in the Symfony ecosystem that helps to spawn a local development environment. It is a lightweight and an alternative way to Docker to run the application locally.
+
+## Prerequisites
+
+- Symfony CLI installed on your machine. You can follow the official [Symfony CLI installation guide](https://symfony.com/download) to install Symfony CLI.
+- PHP Installed locally
+
+Here are various ways to install PHP on your machine. You can choose the one that suits you best.
```

Comment:
can't we link to the requirements page instead? Or move the installation instructions for the requirements here?

Because:
* Currently SQL and node installation instructions are missing here, but you need that as well
* When we change the recommended PHP version, it is likely that this page will be forgotten and the content drift, therefore let's only describe it once